### PR TITLE
[Purview Datamap] Fix anonymous model

### PR DESCRIPTION
**Pull request by catalinaperalta**
Fix anonymous model issue to fix SDK generation.

### CODE REVIEW

**Comment by catalinaperalta** [comment]
.NET failure is due to the single word class names that need to be suppressed in the .NET repo

**Comment by catalinaperalta** [comment]
@yifan-zhou feel free to take over this branch and rename the model to best fit your service. 

@msyyc looks like this should fix the anonymous model issue for multi-part form data.

**Review by msyyc — APPROVED** [review]

**Comment by catalinaperalta** [comment]
Great! Thanks @msyyc :) 

@yifan-zhou922 if the model looks good as is feel free to work with @lirenhe and @msyyc to get the PR merged in during your working hours.

**Review by yifan-zhou922 — APPROVED** [review]

**Review by weidongxu-microsoft — APPROVED** [review]

File: specification/purview/Azure.Analytics.Purview.DataMap/models.tsp [hunk 1]
```diff
@@ -552,6 +552,12 @@ model BulkImportResult {
   successImportInfoList?: ImportInfo[];
 }
 
+@doc("Business metadata to send to the service")
+model BusinessMetadataFile {
```
Comment (weidongxu-microsoft) [inline specification/purview/Azure.Analytics.Purview.DataMap/models.tsp:556]: Maybe a different name?

Java prefer suffix `Options`.

**Review by MaryGao — APPROVED** [review]
Approved from JS side.

**Pull request merged**